src/formik.tsx

```diff
@@ -61,6 +61,8 @@ export interface FormikConfig<
   validateOnChange?: boolean;
   /** Tells Formik to validate the form on each input's onBlur event */
   validateOnBlur?: boolean;
+  /** Tell Formik if initial form values are valid or not on first render */
+  isInitialValid?: (props: Props) => boolean;
```

Comment:
Yeah, I would think so. More common to have a form that needs filling out, than an edit-form that starts life valid.